sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/__init__.py

```diff
@@ -4,5 +4,6 @@
 # ------------------------------------
 from ._models import DeletedSecret, KeyVaultSecret, SecretProperties
 from ._client import SecretClient
+from ._shared._polling import KeyVaultOperationPoller
 
-__all__ = ["SecretClient", "KeyVaultSecret", "SecretProperties", "DeletedSecret"]
+__all__ = ["SecretClient", "KeyVaultSecret", "KeyVaultOperationPoller", "SecretProperties", "DeletedSecret"]
```

Comment:
Perhaps this should have a more descriptive name since we have other Operations which poll and have different semantics in key vault, for instance in certificates. Same with the KeyVaultOperationPoller in azure-keyvault-keys. Perhaps it doesn't matter since these are just return types. Thoughts?

Reply:
we're going to keep the naming the same, but we've gotten rid of references to it in documentation, so users should feasibly never see it and not get confused.